src/components/Search.vue

```diff
@@ -10,17 +10,17 @@
 	const formData: any = ref(searchData.data);
 	const resultOptions = [
 		{ group: 'White Wins', disable: true },
-		{ label: 'X-0', value: 'X-0', description: 'Search by Any Win' },
-		{ label: 'R-0', value: 'R-0', description: 'Search by Road Win' },
-		{ label: 'F-0', value: 'F-0', description: 'Search by Flat Win' },
-		{ label: '1-0', value: '1-0', description: 'Search by Time or Resignation Win' },
+		{ label: 'X-0', value: 'X-0', description: 'Any Win' },
+		{ label: 'R-0', value: 'R-0', description: 'Road Win' },
+		{ label: 'F-0', value: 'F-0', description: 'Flat Win' },
+		{ label: '1-0', value: '1-0', description: 'Default Win' },
```

Comment:
@gruppler I think changing it to default win isn't descriptive enough and could be confused with "any win"